assets/js/modules/analytics/components/settings/SettingsForm.js

```diff
@@ -68,6 +69,14 @@ export default function SettingsForm( {
 		useAnalyticsSnippet ||
 		( useTagManagerSnippet && analyticsSinglePropertyID );
 
+	const gtmContainersResolved = useSelect( ( select ) =>
+		select( MODULES_TAGMANAGER ).hasFinishedLoadingContainers()
+	);
+
+	if ( ! gtmContainersResolved ) {
+		return <ProgressBar />;
+	}
```

Comment:
No need to add the progress bar here. We already have the progress bar in the `SettingsEdit` component which should be updated accordingly.

Reply:
Just read comments left by Nahid and they makes sense to me. So, please, ignore this comment.